academic-services/knowledge-exploration-service/reference-get-histogram.md

```diff
@@ -0,0 +1,223 @@
+---
+title: GET Histogram
+description: Computes attribute value distribution for a structured query expression
+ms.topic: reference
+ms.date: 2020-02-07
+---
+
+# Histogram REST API
+
+The **Histogram** method computes the entities matching a structured query expression and then calculates the distribution of the requested attributes values in the matched entities.
```

Comment:
>structured query expression [](start = 58, length = 27)

for consistency replace with: [structured query expression](concepts-query-expressions.md)